packages/opencensus-core/src/metrics/export/types.ts

```diff
@@ -0,0 +1,340 @@
+ * points.
+ */
+export enum MetricDescriptorType {
+  /** Do not use this default value. */
+  UNSPECIFIED,
+  /** Integer gauge. The value can go both up and down. */
+  GAUGE_INT64,
+  /** Floating point gauge. The value can go both up and down. */
+  GAUGE_DOUBLE,
+  /**
+   * Distribution gauge measurement. The count and sum can go both up and
+   * down. Recorded values are always >= 0.
+   * Used in scenarios like a snapshot of time the current items in a queue
+   * have spent there.
+   */
+  GAUGE_DISTRIBUTION,
+  /**
+   * Integer cumulative measurement. The value cannot decrease, if resets
+   * then the start_time should also be reset.
+   */
+  CUMULATIVE_INT64,
+  /**
+   * Floating point cumulative measurement. The value cannot decrease, if
+   * resets then the start_time should also be reset. Recorded values are
+   * always >= 0.
+   */
+  CUMULATIVE_DOUBLE,
+  /**
+   * Distribution cumulative measurement. The count and sum cannot decrease,
+   * if resets then the start_time should also be reset.
+   */
+  CUMULATIVE_DISTRIBUTION,
+  /**
+   * Some frameworks implemented Histograms as a summary of observations
+   * (usually things like request durations and response sizes). While it
+   * also provides a total count of observations and a sum of all observed
+   * values, it calculates configurable percentiles over a sliding time
+   * window. This is not recommended, since it cannot be aggregated.
+   */
+  SUMMARY
+}
+
+/** Properties of a LabelKey associated with a MetricDescriptor. */
+export interface LabelKey {
+  /** The key for the label. */
+  readonly key: string;
+  /** A human-readable description of what this label key represents. */
+  readonly description: string;
+}
+
+/**
+ * A collection of data points that describes the time-varying values
+ * of a metric.
+ */
+export interface TimeSeries {
+  /**
+   * Must be present for cumulative metrics. The time when the cumulative value
+   * was reset to zero. Exclusive. The cumulative value is over the time
+   * interval (start_timestamp, timestamp]. If not specified, the backend can
+   * use the previous recorded value.
+   */
+  readonly startTimestamp: Timestamp;
+  /**
+   * The set of label values that uniquely identify this timeseries. Applies to
+   * all points. The order of label values must match that of label keys in the
+   * metric descriptor.
+   */
+  readonly labelValues: LabelValue[];
+  /**
+   * The data points of this timeseries. Point.value type MUST match the
+   * MetricDescriptor.type.
+   */
+  readonly points: Point[];
+}
+
+/** Properties of a LabelValue. */
+export interface LabelValue {
+  /** The value for the label */
+  readonly value: string;
+  /**
+   * If false the value field is ignored and considered not set.
+   * This is used to differentiate a missing label from an empty string.
+   */
+  readonly hasValue: boolean;
+}
+
+/** A timestamped measurement. */
+export interface Point {
```

Comment:
The Point should have only timestamp and Value. 

Refer https://github.com/census-instrumentation/opencensus-go/pull/969/files#diff-d6cca479b1a793ab4e0a1a3f22549f84R24 or Java/Python implementation.

Reply:
Done